### ISSUE TITLE
Table headings not converted correctly from odt to latex

### ISSUE THREAD

**Issue by FolkertRA** [jgm/pandoc#3763]
I am converting an ODT document (made in LibreOffice writer) to a LaTeX document. It contains a table with headings. The contents are converted just fine (using version 1.19.2.1, note that this was not working with 1.17.1) but the table heading is completely missing in the output.

To reproduce, start a new empty doc named "test.odt" in LibreOffice writer, add a new table, click the "table heading" checkbox, enter contents into the table. Run "pandoc -f odt -t latex test.odt -o test.tex. The output file does not contain the heading.

**Comment by jgm** [comment]
@MarLinn can you take a look?

**Comment by MarLinn** [comment]
That is correct. Table header elements (both row- and column-headers) are completely ignored right now. They weren't needed and are quite complex to work with if I remember correctly. If you need the contents, put them in the main table area for now.
I would file this as a feature request, not a bug.
